integration/nwo/common/builder.go

```diff
@@ -120,58 +115,13 @@ func (a *artifact) build(args ...string) error {
 }
 
 func (a *artifact) gBuild(input string, args ...string) (string, error) {
-	switch {
-	case strings.HasPrefix(input, "git@"):
-		logger.Infof("building %s", input)
-		// split input in repo#commit#packagePath#cmd
-		entries := strings.Split(input, ";")
-		repo := entries[0]
-		commit := entries[1]
-		packagePath := entries[2]
-		cmd := entries[3]
-
-		testDir, err := os.MkdirTemp("", "fabric")
-		if err != nil {
-			return "", err
-		}
-		logger.Infof("testDir %s", testDir)
-
-		goPath := filepath.Join(testDir, "gopath")
-		logger.Infof("goPath %s", goPath)
-		fabricPath := filepath.Join(testDir, "gopath", "src", packagePath)
-		logger.Infof("fabricPath %s", fabricPath)
-
-		r, err := git.PlainClone(fabricPath, false, &git.CloneOptions{
-			URL:      repo,
-			Progress: os.Stdout,
-		})
-		if err != nil {
-			return "", err
-		}
-		w, err := r.Worktree()
-		if err != nil {
-			return "", err
-		}
-		err = w.Checkout(&git.CheckoutOptions{
-			Hash: plumbing.NewHash(commit),
-		})
-		if err != nil {
-			return "", err
-		}
-		logger.Infof("checked out %s done ", commit)
-
-		logger.Infof("building [%s,%s] ", goPath, packagePath+"/"+cmd)
-
-		return gexec.BuildIn(goPath, packagePath+"/"+cmd)
-	default:
-		if a.raceDetectorEnabled && !strings.HasPrefix(input, "github.com/hyperledger/fabric/") {
-			logger.Infof("building [%s,%s] with race detection ", input, args)
-			return gexec.Build(input, args...)
-		}
-
-		logger.Infof("building [%s,%s] ", input, args)
+	if a.raceDetectorEnabled && !strings.HasPrefix(input, "github.com/hyperledger/fabric/") {
+		logger.Infof("building [%s,%s] with race detection ", input, args)
 		return gexec.Build(input, args...)
 	}
+
```

Comment:
Maybe a check could be added here on what you expect input to be (e.g. not start with git@).

Reply:
no one ever used `git@`, the PR removes completely the support for that.